python/doc/examples/reliability_sensitivity/flood_model.ipynb

```diff
@@ -0,0 +1,300 @@
+    "\n",
+    "It is considered that failure occurs when the difference between the dike height and the water level is positive. In other words, we want to estimate the probability:\n",
+    "\n",
+    "$$\n",
+    "P_f = P(S>0).\n",
+    "$$\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## References\n",
+    "\n",
+    "* Iooss B, Lemaître P (2015) A review on global sensitivity analysis methods. In: Meloni C., Dellino G. (eds) Uncertainty management in Simulation-Optimization of Complex Systems: Algorithmsand Applications, Springer\n",
+    "* Baudin M., Dutfoy A., Iooss B., Popelin AL. (2015) OpenTURNS: An Industrial Software for Uncertainty Quantification in Simulation. In: Ghanem R., Higdon D., Owhadi H. (eds) Handbook of Uncertainty Quantification. Springer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define the model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from __future__ import print_function\n",
+    "import openturns as ot"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the marginal distributions of the parameters\n",
+    "dist_Q = ot.TruncatedDistribution(ot.Gumbel(558., 1013.), 0, ot.TruncatedDistribution.LOWER)\n",
+    "dist_Ks = ot.TruncatedDistribution(ot.Normal(30.0, 7.5), 0, ot.TruncatedDistribution.LOWER)\n",
+    "dist_Zv = ot.Uniform(49.0, 51.0)\n",
+    "dist_Zm = ot.Uniform(54.0, 56.0)\n",
+    "marginals = [dist_Q, dist_Ks, dist_Zv, dist_Zm]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the Copula\n",
+    "RS = ot.CorrelationMatrix(4)\n",
+    "#RS[2, 3] = -0.2\n",
+    "# Evaluate the correlation matrix of the Normal copula from RS\n",
+    "R = ot.NormalCopula.GetCorrelationFromSpearmanCorrelation(RS)\n",
+    "# Create the Normal copula parametrized by R\n",
+    "copula = ot.NormalCopula(R) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the joint probability distribution\n",
+    "distribution = ot.ComposedDistribution(marginals, copula)\n",
+    "distribution.setDescription(['Q', 'Ks', 'Zv', 'Zm'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create the model\n",
+    "model = ot.SymbolicFunction(['Q', 'Ks', 'Zv', 'Zm'],\n",
+    "                            ['(Q/(Ks*300.*sqrt((Zm-Zv)/5000)))^(3.0/5.0)+Zv-55.5-3.'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define the event"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then we create the event we want to estimate the probability."
```

Comment:
```suggestion
    "Then we create the flood event."
```

Reply:
Fixed. I also removed the copula which was set for no reason.